index.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>    
+    <nav id="navbar">
+        <a href="#" class="logo">My Logo</a>
+        <ul>
+            <li><a href="#portfolio" class="nav-links">Portfolio</a></li>
+            <li><a href="#About" class="nav-links">About</a></li>
+            <li><a href="#contact" class="nav-links">Contact</a></li>
+        </ul>
+      </nav>
```

Comment:
- [ ] Kindly **as suggested by the previous reviewer 🙏** It would be better to keep the mobile version of your page identical to that of the Figma design 😎
    - You may try to replicate the navigation bar to match that of the Figma design and for the best user experience 👍 
      ![image](https://user-images.githubusercontent.com/102757126/236323570-5e02eb40-0e19-4328-8700-d085ff469c9e.png)

    - According to the Figma design, all the project cards should have the `image` on the top and the `project description` after, But they are currently switching the order to that of the Desktop version 😊😎
   ![image](https://user-images.githubusercontent.com/102757126/236323675-52b561fc-3214-4be9-9974-6c2aac9c8dd7.png)

      #### KEEP IT UP 👏 👏

Reply:
- [ ] Great work 👍🏾